### ISSUE TITLE
Does not work with Flutter 0.6.0

### ISSUE THREAD

**Issue by agrosner** [xqwzts/flutter_circular_chart#14]
When upgrading the library to flutter 0.6.0, the library fails to solve versions:
```
The current Dart SDK version is 2.1.0-dev.0.0.flutter-be6309690f.

Because flutter_circular_chart 0.0.3 requires SDK version >=1.19.0 <2.0.0 and no versions of flutter_circular_chart match >0.0.3 <0.1.0, flutter_circular_chart ^0.0.3 is forbidden.
So, because flutter_example depends on flutter_circular_chart ^0.0.3, version solving failed.
pub get failed (1)

```

**Comment by agrosner** [comment]
Added a merge request to fix this issue:
https://github.com/xqwzts/flutter_circular_chart/pull/15

**Comment by xqwzts** [comment]
@agrosner  Thanks for reporting and your effort in the PR.

Fixed in #17
